config/initializers/assets.rb

```diff
@@ -9,4 +9,4 @@
 # Precompile additional assets.
 # application.js, application.css, and all non-JS/CSS in the app/assets
 # folder are already added.
-# Rails.application.config.assets.precompile += %w( admin.js admin.css )
+Rails.application.config.assets.precompile += %w[bootstrap.min.js]
```

Comment:
This is where the error message told me to put this. It could also go in the `manifest.js` instead. Not sure which is better.